ckanext/cnra_schema/templates/scheming/package/snippets/package_form.html

```diff
@@ -1,9 +1,53 @@
 {% ckan_extends %}
 
+{% block basic_fields %}
+  {% block styles %}
+      <link rel="stylesheet" href="/css/additional_metadata_form.css" />
+  {% endblock%}
+
+  <p class=right-text>
+    Questions? See: <a href="/pages/metadata"><u>Metadta Guide</u></a>
+  </p>
+  
+  {%- if not dataset_type -%}
+    <p>
+    dataset_type not passed to template. your version of CKAN
+    might not be compatible with ckanext-scheming
+    </p>
+  {%- endif -%}
+  {%- set schema = h.scheming_get_dataset_schema(dataset_type) -%}
+  <!-- This for loop is intended to separate primary metadata and additiooanl metadata-->
```

Comment:
Please update this comment to state that the loop is to display primary metadata.